plugins/woocommerce/includes/admin/settings/class-wc-settings-payment-gateways.php

```diff
@@ -193,7 +193,7 @@ public function payment_gateways_setting() {
 											echo '<a class="button alignright" aria-label="' . esc_attr( sprintf( __( 'Manage the "%s" payment method', 'woocommerce' ), $method_title ) ) . '" href="' . esc_url( admin_url( 'admin.php?page=wc-settings&tab=checkout&section=' . strtolower( $gateway->id ) ) ) . '">' . esc_html__( 'Manage', 'woocommerce' ) . '</a>';
 										} else {
 											if (
-												'WooCommerce Payments' === $method_title &&
+												'WooPayments' === $method_title &&
```

Comment:
Just checking - where does `$method_title` originally come from? Will this check potentially cause issues with past versions of WCPay?

It may be worth doing something like:

```php
( 'WooCommerce Payments' === $method_title || 'WooPayments' === $method_title )
```

Just to be on the safe side. 🙂

Reply:
I agree! Done.